bluesky/plans.py

```diff
@@ -570,7 +570,7 @@ def tune_centroid(
         detectors, signal, motor,
         start, stop, min_step,
         num=10,
-        step_factor=2,
+        step_factor=3.0,
```

Comment:
why is this 3 by default? mainly curious. seems like it would be 4 from doc?

Reply:
Choice of 3 is arbitrary.  Balances between rapid convergence and possibly missing a low S/N peak.

Seems as if the upper bound should be `num`.  Agree?

Reply:
Definitely tested in simulation.  Here's *the* [unit test for `tune_centroid()`](https://github.com/prjemian/bluesky/blob/master/bluesky/tests/test_scans.py#L280):

Reply:
great. I'm wondering if we should include a test for `step_factor` at least. Maybe at least something like:
```python
    with pytest.raises(ValueError):  
        # assert raises ValueError if step_factor < 1
        scan5 = bp.tune_centroid([det], 'det', motor, 5, 0, -0.1, 10, 
                                 step_factor=.5, snake=False)
        RE(scan5)
```
could not hurt to add in `test_tune_centroid`